airflow/api_fastapi/core_api/routes/public/assets.py

```diff
@@ -74,6 +84,56 @@ def get_assets(
     )
 
 
+@assets_router.get(
+    "/events",
+    responses=create_openapi_http_exception_doc([401, 403, 404]),
```

Comment:
401, and 403 have been added to the root router, all routes now inherit that and we don't need to specify it anymore. Just other errors such as 404 need to be specified.